homeassistant/components/cambridge_audio/entity.py

```diff
@@ -24,3 +49,17 @@ def __init__(self, client: StreamMagicClient) -> None:
             serial_number=client.info.unit_id,
             configuration_url=f"http://{client.host}",
         )
+
+    @callback
```

Comment:
Please remove `@callback`.

Reply:
Addressed in https://github.com/home-assistant/core/pull/126082